packages/mongodb-rag-ingest/src/embed/updateEmbeddedContent.ts

```diff
@@ -37,26 +41,29 @@ export const updateEmbeddedContent = async ({
       sourceNames ? ` in sources: ${sourceNames.join(", ")}` : ""
     }`
   );
-  for (const page of changedPages) {
-    switch (page.action) {
-      case "deleted":
-        logger.info(
-          `Deleting embedded content for ${page.sourceName}: ${page.url}`
-        );
-        await embeddedContentStore.deleteEmbeddedContent({
-          page,
-        });
-        break;
-      case "created": // fallthrough
-      case "updated":
-        await updateEmbeddedContentForPage({
-          store: embeddedContentStore,
-          page,
-          chunkOptions,
-          embedder,
-        });
-    }
-  }
+  await PromisePool
+    .withConcurrency(embedConcurrencyOptions?.processPages || 1)
```

Comment:
```suggestion
    .withConcurrency(embedConcurrencyOptions?.processPages ?? 1)
```

prefer [nullish coalescing (`??`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) instead of or operating (`||`)